.eslintrc

```diff
@@ -1,25 +1,55 @@
 {
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "requireConfigFile": false,
+    "ecmaFeatures": {
+      "experimentalObjectRestSpread": true
+    }
+  },
   "env": {
     "browser": true,
     "commonjs": true,
     "es6": true,
-    "jquery": true
+    "jquery": true,
+    "jest/globals": true
   },
   "extends": [
     "standard",
-    "standard-jsx"],
+    "standard-jsx",
+    "plugin:jsx-a11y/recommended",
+    "plugin:react/recommended",
+    "plugin:import/recommended",
+    "plugin:n/recommended"
+  ],
   "rules": {
-    "jsx-quotes": [2, "prefer-double"]
+    "jsx-quotes": [2, "prefer-double"],
+    "jsx-a11y/no-onchange": "off",
+    "react/prop-types": "off",
+    "n/no-missing-require": "off",
+    "n/no-unsupported-features/es-syntax": "off",
+    "n/no-missing-import": "off",
+    "n/no-unpublished-import": "off",
+    "n/no-extraneous-import": "off"
```

Comment:
is it recommended to switch off these rules or is it just because our code doesn't work when they are on? :p

Reply:
ok saw the answer in a+:)